src/handlers/handle_input.py

```diff
@@ -53,7 +56,18 @@ def handle_input(
                 # KeyboardInterrupt was triggered and None was returned so exit
                 return
 
-        match input_command:
+        match input_command:            
+            case "server":
+                match input_selected_object:
+                    case "is":
```

Comment:
I don't like the server check with `server is` currently. Maybe change it to `server get` or something?